src/Benchmarks/FortunesEfMiddleware.cs

```diff
@@ -44,9 +44,7 @@ public FortunesEfMiddleware(RequestDelegate next)
 
         private static async Task<IEnumerable<Fortune>> LoadRows(ApplicationDbContext dbContext)
         {
-            var result = new List<Fortune>();
-
-            result.AddRange(await dbContext.Fortunue.ToListAsync());
+            var result = await dbContext.Fortune.ToListAsync();
```

Comment:
Perfect, thanks! Safe to make that change then :smile:

Did you want to fix this PR up so I can merge or should I just make the change?